GENERATORS/SIMULATIONS/RAND/RAND.py

```diff
@@ -45,7 +45,13 @@ def RAND(
 
     seed = random.randint(1, 10000)
     my_generator = np.random.default_rng(seed)
-    size = len(default.x) if default is not None else 1
+
+    match default:
```

Comment:
`size` should be `None` if `default` is `None`, if you pass `size=1` to numpy's random generator then it gives an array with 1 element in it instead of a scalar